pkgs/swift2objc/lib/src/generate_wrapper.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+import '../swift2objc.dart';
+import 'config.dart';
+import 'generator/generator.dart';
+import 'parser/parser.dart';
+import 'transformer/transform.dart';
+
+Future<void> generateWrapper(Config config) async {
+  final parentDir = config.tempDir ?? defaultTempDir;
+  final tempDir = await parentDir.createTemp(defaultTempDirPrefix);
+
+  await _generateSymbolgraphJson(
+    config.inputFiles,
+    tempDir,
+  );
+
+  String? symbolgraphJsonPath;
+  await for (final entity in tempDir.list()) {
```

Comment:
well this situation shouldn't happen cause `createTemp` creates a new directory with a random suffix every time. But still it would be better to know the name of the Symbolgraph beforehand. I just found that passing multiple files results in a different name so I wanted to make sure to always find the symbolgraph json. I'll add '-module-name' flag to the `swiftc` command.